build/circle-test.sh

```diff
@@ -95,7 +100,12 @@ time ${builder} make GITHOOKS= testrace \
 # acceptance tests on the Mac's, but circle-deps.sh only built the
 # acceptance tests for Linux.
 if [ "$(uname)" = "Linux" ]; then
-  time $(dirname $0)/../acceptance.test -test.v -test.timeout 5m
+  time $(dirname $0)/../acceptance.test -test.v -test.timeout 5m --verbosity=1 --vmodule=monitor=2 | \
+    tr -d '\r' | tee "${outdir}/acceptance.log" | \
+    grep -E "^\--- (PASS|FAIL)|^(FAIL|ok)|${match}" |
+    awk '{print "acceptance:", $0}'
+  echo 'FAIL github.com/cockroachdb/cockroach/acceptance 1337s' \
```

Comment:
Nah, this is required. Because we are not running `go test` here, we don't get the package-level summary line, which breaks go2xunit. This line fakes that.